Possible issue with building Ubuntu 20.04

I'm following `docs/INSTALL.md` on Ubuntu 20.04, but the build gets stuck.

The first problem I ran into (which may be related to the latter) was that the documentation gives the following instructions for installing build dependencies:

```
sudo apt-get update && sudo apt-get install -y \
    build-essential \
    cmake \
    clang \
    ninja-build \
    python3 \
    qtbase5-dev \
    libc++-dev \
    libfreetype-dev \
    libglu1-mesa-dev \
    libicu-dev \
    libqt5svg5-dev \
    libsqlite3-dev \
    zlib1g-dev
```

(Ubuntu 20.04 is explicitly mentioned.)

This however installs Cmake version `3.16.*`, but the build requires `3.18.0` at the least. I've installed version `3.23.2` manually from the Cmake website. So this may be a bug in the documentation.

After that, the build (`tools/unix/build_omim.sh -r desktop`) starts, but I get stuck somewhere in the part where Unity is built. I've attached the output. Any idea what I'm doing wrong? Is Ubuntu 20.04 still supported as build environment?
[build-output.txt](https://github.com/organicmaps/organicmaps/files/8834868/build-output.txt)

@vng Patch applies cleanly. I get a different output, but this error at the patched line:

```
In file included from /home/freaknl/Dev/osm/omim-build-release/generator/CMakeFiles/generator.dir/Unity/unity_0_cxx.cxx:99:
/home/freaknl/Dev/osm/organicmaps/generator/intermediate_data.cpp:67:11: error: non-deleted function ‘virtual generator::cache::{anonymous}::PointStorageWriterBase::~PointStorageWriterBase()’ overriding deleted function
   67 |   virtual ~PointStorageWriterBase() noexcept(false) override = default;
```
[build-output-patched.txt](https://github.com/organicmaps/organicmaps/files/8839523/build-output-patched.txt)

--- a/generator/intermediate_data.hpp
+++ b/generator/intermediate_data.hpp
@@ -58,7 +58,7 @@ static_assert(std::is_trivially_copyable<LatLonPos>::value, "");
 class PointStorageWriterInterface
 {
 public:
-  virtual ~PointStorageWriterInterface() noexcept(false) = default;
+  virtual ~PointStorageWriterInterface() noexcept(false) {};
   virtual void AddPoint(uint64_t id, double lat, double lon) = 0;
   virtual uint64_t GetNumProcessedPoints() const = 0;
 };

